pkg/bffauth/challenge_test.go

```diff
@@ -0,0 +1,41 @@
+package bffauth
+
+import (
+	"testing"
+
+	"github.com/bluesky-social/indigo/atproto/crypto"
+)
+
+func TestChallengeProofFlow(t *testing.T) {
+	// Generate a test key pair
+	privateKey, err := crypto.GeneratePrivateKeyP256()
+	if err != nil {
```

Comment:
same here let's standardize testing
I would go with `require.NoError(t, err, "failed to generate key pair")` here -- require does the same as fatal and then you don't need `err != nil`. i think this is one of the standard packages.